tests/IntegrationTests/Monai.Deploy.WorkflowManager.WorkflowExecutor.IntegrationTests/Features/TaskDestinations.feature

```diff
@@ -41,3 +41,55 @@ Scenario: Publish a valid Task Update event as failed which does not trigger a n
     When I publish a Task Update Message Task_Update_Dispatches_Single_Task with status Failed
     Then A Task Dispatch event is not published
     And Workflow Instance status is Failed
+
+@TaskUpdate
```

Comment:
Could you tag them with @TaskDestinationConditions? @TaskUpdate seem very generic...we should have a look at the others as well but can do that as part of a clean up :)

Reply:
Added